### ISSUE TITLE
Don't reply to retweets

### ISSUE THREAD

**Issue by matthewberryman** [smart-facility/cognicity-reports-powertrack#26]
Unless quoted

**Comment by matthewberryman** [comment]
The problem is that with the [proposed changes](https://blog.twitter.com/2016/doing-more-with-140-characters), tweets that start with an '@' but that aren't replies will be seen to all followers of the account that sent the tweet. In the bot at present, it also "replies" to people who've retweeted a tweet, but merely tags them without setting the reply metadata up as it can't in that scenario. In that case, once the changes are made by Twitter, followers of the account will see a lot of "spam".

Also, people who are retweeting a tweet are most likely not directly reporting flooding affecting them anyway.

**Comment by benatwork99** [comment]
I committed to a branch 'retweet' what should be a fix for this, but did not have time to test this at all (which is why I didn't commit to the refactor branch).

Note that the 'retweet' branch was forked from the 'refactor' branch and CI tests are currently failing because the 'master' branch of cognicity-reports doesn't have the base class refactoring that's expected by the code in this branch. I.e., CI will pass once 'refactor' is merged into master on cognicity-reports.

**Comment by benatwork99** [comment]
I committed to a branch 'retweet' what should be a fix for this, but did not have time to test this at all (which is why I didn't commit to the refactor branch).

Note that the 'retweet' branch was forked from the 'refactor' branch and CI tests are currently failing because the 'master' branch of cognicity-reports doesn't have the base class refactoring that's expected by the code in this branch. I.e., CI will pass once 'refactor' is merged into master on cognicity-reports.
